vue.config.js

```diff
@@ -11,9 +11,6 @@ except according to the terms contained in the LICENSE file.
 */
 module.exports = {
   chainWebpack: (config) => {
-    // TODO/requestData. Revisit this.
-    config.resolve.alias.set('pinia', 'pinia/dist/pinia.cjs');
```

Comment:
Previously, I was having trouble using Pinia without this line, but everything seems fine now without it (building, linting, testing). We don't use Pinia in production, only development, so it feels OK to make this change now.